core/src/saros/negotiation/AdditionalProjectDataFactory.java

```diff
@@ -22,6 +22,7 @@
  * providers are disjoint. If multiple providers use the same key, the mapping of the provider added
  * last will be used.
  */
+// TODO remove as no longer used?
```

Comment:
Extend explanation and/or link to PR.

Reply:
I decided to remove this logic in a separate PR as it is a change to the core. Furthermore, changing it here would cause merge conflicts down the line when rebasing the core cleanup (#1012).